src/pcdns.c

```diff
@@ -493,8 +493,7 @@ static int lookup_domain (
        BOOL              is_aaaa,       /* Query A or AAAA record */
        void             *addr)          /* return address */
 {
-  _udp_Socket sock;
-  sock_type  *dom_sock = NULL;
+  sock_type  sock, *dom_sock = NULL;
```

Comment:
Please note: I looked but couldn't find anywhere in the function that sock is actually assigned a value.